debian/control

```diff
@@ -44,7 +44,7 @@ Recommends: cryptsetup,
             mdadm (>> 4.1~rc1-2),
             parted,
             python3-progressbar,
-            ttf-dejavu,
+            fonts-dejavu,
```

Comment:
do you by chance know how far back it's been `fonts-dejavu`?   Just wondering if it makes sense to be
`fonts-dejavu | ttf-dejavu`

Reply:
fonts-dejavu was available in trusty, possibly before.

this is the longest/oldest package name transition ever to date.

Reply:
wow, thanks,  OK merging.